02.calendar/cal.rb

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'optparse'
+require 'date'
+
+TARGET_MONTH_BLANK_SPACE = 15
+DATE_BLANK_SPACE = 2
+
+options = {}
+
+opt = ARGV.getopts('m:', 'y:')
+options[:month] = opt['m'].to_i if opt['m']
+options[:year] = opt['y'].to_i if opt['y']
+options[:month] = opt['m']&.to_i || Date.today.month
+options[:year] = opt['y']&.to_i || Date.today.year
+
+puts "#{options[:month]}月 #{options[:year]}年".rjust(TARGET_MONTH_BLANK_SPACE)
```

Comment:
```suggestion
puts "     #{options[:month]}月 #{options[:year]}年"
```

みたいな感じで固定のスペースを左に詰める、とかでも良いかと思います。

Reply:
ありがとうございます！
固定のスペースを詰める方がシンプルですし、定数名も不要になるので良いですね。
固定のスペースを入れて、`cal`コマンドと同じ位置に表示されるように調整しました。